content/influxdb/v2.0/upgrade/v1-to-v2/_index.md

```diff
@@ -132,10 +132,14 @@ to scrape data from the `/metrics` endpoint and store them in a bucket.
 
 InfluxDB 2.0 requires authentication and does not support the InfluxDB 1.x
 `auth-enabled = false` configuration option.
-Consider [enabling authentication in your InfluxDB 1.x instance](/influxdb/v1.8/administration/authentication_and_authorization/#set-up-authentication)
+You should [enable authentication in your InfluxDB 1.x instance](/influxdb/v1.8/administration/authentication_and_authorization/#set-up-authentication)
 **before** upgrading to InfluxDB 2.0 to ensure the appropriate credentials are in place and
 that the various applications, agents, and visualization tools are able to connect.
 
+If you attempt to upgrade with `auth-enabled = false` in your 1.x configuration, a warning is produced.
+If you see this warning, it is recommended to reconfigure your 1.x instance with `auth-enabled = true`
+and ensure all of the username and password combinations are working as expected before retrying the upgrade process.
```

Comment:
Before upgrading to 2.0, [enable....]() to ensure your applications, agents, and visualization tools have authenticated credentials to connect to InfluxDB. If you upgrade with `auth-enabled = false`, the upgrade may appear complete, but client requests to InfluxDB 2.0 may be silently ignored (you won't see a notification the request was denied).

Reply:
yep  ^^.  Perfect.